test/static/es6-support.js

```diff
@@ -46,3 +47,82 @@ class A {
     this.data = 'Hello World';
   }
 }
+
+/**
+ * I am a description of test-element.
+ *
+ * @hero /path/to/hero.png
```

Comment:
I'm just confused why this element is being added. This style of element was already supported, right?